scripts/DllManipulator.cs

```diff
@@ -123,6 +126,8 @@ public static void Reset()
             _customLoadedTriggers?.Clear();
             _customAfterUnloadTriggers?.Clear();
             _customBeforeUnloadTriggers?.Clear();
+
+            Options = null;
```

Comment:
Wait a sec, what does it do?

Reply:
We delete the DllManip Options copy. This is also used as an indicator to whether the DllManip is initialized.

Reply:
I mean, how does it consume the options when they are null? Sry if I miss sth

Reply:
Ah nvm, on the diff it blended in as though it was called in Initalize, not Reset, sorry